src/main/kotlin/no/nav/arrangor/arrangor/ArrangorControllerServiceUser.kt

```diff
@@ -18,4 +19,10 @@ class ArrangorControllerServiceUser(
 	fun getArrangor(@PathVariable("orgnummer") orgnummer: String): ArrangorMedOverordnetArrangor {
 		return arrangorService.getArrangorMedOverordnetArrangor(orgnummer)
 	}
+
+	@GetMapping("{id}")
+	fun get(@PathVariable("id") id: UUID): ArrangorMedOverordnetArrangor {
```

Comment:
Det har vel ikke så mye å si, men siden vi har hatt sånn fokus på tokenvalidering i det siste, så mangler vel denne controlleren en sjekk på at det er et M2M token, hvis det er det den skal ta i mot?

Reply:
Jeg tror ikke det er nødvendig her siden det kun er amt-tiltak og bff-en som har tilgang til apiene her. bff-en støtter kun tokenx, så i praksis kan kallene kun komme fra amt-tiltak til disse apiene. Vi kan gjerne legge det på senere da, bare for å ha det :)